pyresample/ewa/ewa.py

```diff
@@ -149,7 +149,7 @@ def fornav(cols, rows, area_def, data_in,
     """
     if isinstance(data_in, (tuple, list)):
         # we can only support one data type per call at this time
-        assert(in_arr.dtype == data_in[0].dtype for in_arr in data_in[1:])
+        assert (in_arr.dtype == data_in[0].dtype for in_arr in data_in[1:])
```

Comment:
```suggestion
        for in_arr in data_in[1:]:
           if in_arr.dtype != data_in[0].dtype:
               raise ValueError("All input arrays must be the same dtype")
```

Reply:
I think we're supposed to avoid asserts in non-test code so I figured since you were updating it anyway why don't I just remove the assert.

Reply:
Oh yeah, I was just fixing the emerging new flake8 warnings. Thanks!